src/openforms/submissions/templates/admin/submissions/submission/logs.html

```diff
@@ -17,21 +17,34 @@
 {% endblock %}
 
 {% block content %}
-    <h1>{% trans  "Logic logs" %}</h1>
     {% block field_sets %}
         {% if not logs_activity %}
             {% trans "No logic rules for that submission" %}
         
         {% else %}
             {% for log in logs_activity  %}
-                <span class="timestamp-log"></span>
                 <fieldset class="module aligned collapse collapsed fieldset-log">
                     <h2>
                         <time datetime="{{ log.timestamp.isoformat }}" title="{{ log.timestamp|date:'Y-m-d H:i:s' }}">
                             {% blocktrans with since=log.timestamp|timesince %}{{ since }} ago{% endblocktrans %}
                         </time>
                         - {% trans 'Logic' %} -
                     </h2>
+                    <h4> {% trans "Variables" %} </h4>
+                    <table class="logic-rules-tables">
+                        <thead>
+                            <th class="property-name-width"> {% trans "Property name" %} </th>
+                            <th> {% trans "Value" %} </th>
+                        </thead>
+                        <tbody>
+                            {% for key, value in log.extra_data.involved_components.items  %}
+                            <tr>
+                                <td>{{ value.stepName }}: <code>{{ key }}</code> &#40;{{ value.label }}&#41;</td>
+                                <td>{{ value.actualValue }}</td>
+                            </tr>
+                            {% endfor %}
+                        </tbody>
+                    </table>
```

Comment:
Now is the time to talk about BEM :)

The rough strokes of this article are good to familiarize yourself: https://css-tricks.com/bem-101/

Summarized - the idea is that you split your styling based on the idea of **blocks** and **elements** (the B and E from BEM). Elements only exist in the context of their parent block. With BEM, this means you use classnames in the form of `.block` and `.block__element`. One block can have many different elements, and markup-wise, they may even be nested. You cannot have elements of elements (meaning `.block__element__other-element` is illegal.

Modifiers apply to both blocks and elements, and are essentially used to implement variations of the same thing.

Here, your tables act as blocks, and I would use a class name that appropriately reflects that: `.logic-logs`. I see two variants: a table for input variables, and a table for evaluated rules, so in the markup I'd expect something like:

```html
<table class="logic-logs logic-logs--variables">
    ...
</table>

<table class="logic-logs logic-logs--rules">
    ...
</table>
```
We've applied the modifiers to the block so that we can target the styling appropriately.


Next, you have your header rows/columns - those only make sense in the context of their parent table, so they should be elements. I'd suggest something like `<th class="logic-logs__col-header">...</th>`. You can see the element syntax (double underscore).

Finally, since the different types of tables require different widths, we can target those using modifiers, e.g.:

```html
<table class="logic-logs logic-logs--variables">
    <thead>
        <th class="logic-logs__col-header logic-logs__col-header--name">{% trans "Property name" %}</th>
        <th class="logic-logs__col-header logic-logs__col-header--value">{% trans "Value" %}</th>
    </thead>
</table>

<table class="logic-logs logic-logs--rules">
    <thead>
        <th class="logic-logs__col-header logic-logs__col-header--rule"> {% trans "Rule" %} </th>
        <th class="logic-logs__col-header logic-logs__col-header--trigger"> {% trans "Triggered" %} </th>
        <th> {% trans "Action" %} </th>
        <th></th>
    </thead>
</table>
```

You would then write some SCSS in the form of:

```scss
.logic-logs {
  width: 100%;

  &--variables {
    .logic-logs__col-header {
      &--name {
        width: 70%;
      }
    }
  }

  &--rules {
    .logic-logs__col-header {
      &--rule { 
        width: 60%;
      }
      &--trigger {
        width: 10%;
      }
    }
  }
}
```

the net result is that your styles are fairly well isolated and scoped within the parent `.logic-logs` block.